src/test/scala/org/broadinstitute/dsde/workbench/sam/db/TestDbReference.scala

```diff
@@ -0,0 +1,100 @@
+package org.broadinstitute.dsde.workbench.sam.db
+
+import cats.effect.{IO, Resource}
+import com.google.common.base.Throwables
+import com.typesafe.config.ConfigFactory
+import com.typesafe.scalalogging.LazyLogging
+import io.opencensus.trace.AttributeValue
+import liquibase.database.jvm.JdbcConnection
+import liquibase.resource.{ClassLoaderResourceAccessor, ResourceAccessor}
+import liquibase.{Contexts, Liquibase}
+import org.broadinstitute.dsde.workbench.sam.config.LiquibaseConfig
+import org.broadinstitute.dsde.workbench.sam.db.DatabaseNames.DatabaseName
+import org.broadinstitute.dsde.workbench.sam.db.TestDbReference.databaseEnabled
+import org.broadinstitute.dsde.workbench.sam.util.SamRequestContext
+import org.broadinstitute.dsde.workbench.util2.ExecutionContexts
+import scalikejdbc.config.DBs
+import scalikejdbc.{ConnectionPool, DBSession, IsolationLevel}
+
+import java.security.cert.CertPathBuilderException
+import java.sql.SQLTimeoutException
+import scala.concurrent.ExecutionContext
+
+object TestDbReference extends LazyLogging {
+  private val config = ConfigFactory.load()
+  private val databaseEnabled = config.getBoolean("db.enabled")
+
+  // This code is copied over from DbReference. We didnt want to have to refactor DbReference to be able to use it in tests.
```

Comment:
We added this line in the mob. Does this sound right to you?

Reply:
Yes!